samples/delete_deployment.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020, Google LLC.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/**
+ * Delete a Game Servers Deployment.
+ * @param {string} projectId string project identifier
+ * @param {string} deploymentId unique identifier for the new Game Server Deployment
+ */
+function main(
+  projectId = 'YOUR_PROJECT_ID',
+  deploymentId = 'DEPLOYMENT_ID'
+) {
+  // [START cloud_game_servers_delete_deployment]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'Your Google Cloud Project ID';
+  // const deploymentId = 'A unique ID for the Game Server Deployment';
+  const {
+    GameServerDeploymentsServiceClient,
+  } = require('@google-cloud/game-servers');
+
+  const client = new GameServerDeploymentsServiceClient();
+
+  async function deleteGameServerDeployment() {
+    const request = {
+        // The full resource name
+        name: `projects/${projectId}/locations/global/gameServerDeployments/${deploymentId}`,
```

Comment:
@alexander-fenster this is less than ideal. Do we have a tracking bug in gax/generator to provide a helper method for these paths?

Reply:
It's there! https://github.com/googleapis/nodejs-game-servers/blob/master/src/v1beta/game_server_deployments_service_client.ts#L175

To use it: `client.gameServerDeploymentPath(projectId, 'global', deploymentId)`

Reply:
note: this method isn't going to work for rollouts, use the gameServerDeploymentPath for the rollout's name field.  This is a known issue filed internally.

Reply:
We are missing a helper for `projects/${project}/locations/${location}`, the tracking bug is https://github.com/googleapis/gapic-generator-typescript/issues/570. But the one you mentioned exists :)

Reply:
Thank you :)

Reply:
Thanks for the clarification. Done!